ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+* Read the ITE to check if it is ready, sound, and complete. The ideas
+ must make technical sense, even if they don't seem likely to be
+ accepted.
+
+* The title should accurately describe the content.
+
+* Edit the ITE for minor non-controversial language
+(spelling, grammar, sentence structure, etc.), markup, code style changes.
+For significant or time consuming changes,
+the editor may choose to provide feedback instead.
+
+==== Request Changes
+
+If the ITE isn't ready, an editor will send it back to the sponsor for
+revision with specific instructions.
+
+==== Approve as Draft
+
+Once the ITE is ready for the repository, a ITE editor will:
+
+. Assign a ITE number (almost always just the next available number, but
+  may also be a special/joke number, like 666 or 3141).
+. Update the folder number to match the ITE number
+. Update the ITE number in the document.
+. Update the ITE status using the to draft.
+. Commit all changes and push them to the branch in the PR
+. "Squash and merge" the PR into the `master` branch.
+
+==== Permission group membership
+
+Editors add and remove Sponsors from the appropriate permission groups on the
+ITE repository. When a ITE includes a new Sponsor, an editor will add that
+person to the "ITE Sponsors" or GitHub group respectively. When someone is no
+longer an active Sponsor, a ITE editor will remove them from the permission
+group.  Editors will clean up the the permission groups from time to time as
+they see the need or are asked to do so.
+
+== Motivation
+
+in-toto has classically been driven by "you-had-to-be-there" development. With
+specific changes largely being driven by smaller independent groups of
+developers (sometimes just one). Sometimes, decisions were made and not
+properly documented, which resulted in additional overhead and mind-reading
+efforts to dig long-established rationales.
+
+Design documents extending back into the history of in-toto are few and far
+between, as the project grew organically over time. As such, a contributor,
+existing or future, must read mountains of code, pull requests, mailing list
+discussions, etc, in order to fully understand how/what/why for many major
+subsystems, design decisions and already-explored-paths within the in-toto
+framework.
+
+The in-toto Enhancement aims to address both of these major issues by
+providing an understood process for making sizable, but understandable,
+enhancements to in-toto.
+
+=== Benefits to existing developers
+
+ITE provides a systematic approach for vetting and developing new proposals and
+ideas for in-toto. By encouraging "everybody to follow the rules" it will be
+easier for existing developers to get their ideas and changes into in-toto 
+without finding themselves mired in unspoken cultural norms within the project.
+
+=== Benefits to future developers
+
+By providing clear, understandable, and bite-sized design documents which would
+explain various subsections of in-toto. ITEs also make it clearer how an
```

Comment:
Change dot (.) to comma (,) before "ITEs"?

Reply:
I'm not sure about this one. Care to elaborate about the rationale ? :)

Reply:
"By providing clear, understandable, and bite-sized design documents which would
 explain various subsections of in-toto." sounds like an imcomplete sentence. Something it's missing.

Reply:
ah, let me clean this up